README.md

```diff
@@ -48,32 +35,15 @@ So that I am reassured that my order will be delivered on time
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
-
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
-
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
-
+This project attempts to recreate a takeaway application to be able to place an order and receive a txt message for confirmation.
+This project is not completed and attempted until user story 3 with a minor bug during the calculation of the total price, yet to be resolved.
 
-In code review we'll be hoping to see:
+The design of the application was made using sequence diagrams and class diagrams which allowed me to create 3 classes bearing in mind the single responsibility principle.
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+Initially a takeaway class was created which delegated the responsibility to print the menu to a menu class.
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+The menu class would hand the calculation of the final price to a order class.
 
-Notes on Test Coverage
-------------------
+In the order class there is a bug that needs to be resolved as the price calculation it currently returns nil.
 
-You can see your [test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) when you run your tests.
+A combination of OOP techniques was used such as Dependency injection, methods Stubbing and Mocking
```

Comment:
Would like to see an example of how to interact with the code and run the tests, but the README does include good information about your approach and progress.